docs/README.md

```diff
@@ -1,18 +1,51 @@
-📢 Don't fork this project. Use, contribute or open issues in<a href="https://github.com/vtex-apps/store-discussion"> Store Discussion</a>
+📢 Use this project, [contribute](https://github.com/vtex-apps/product-customizer) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
+    
+# Product Customizer
+
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-    
-# Product Customizer
 
 The Product Customizer allows a product's [attachments](https://help.vtex.com/tutorial/o-que-e-um-anexo--aGICk0RVbqKg6GYmQcWUm) to be made available and ready to be chosen by users from the product page. 
 
 ![attachment-product-customization-select](https://user-images.githubusercontent.com/52087100/65711995-707f6e00-e06c-11e9-8faa-43aecfed3e51.png)
 
 ## Configuration 
 
-1. Add the `product-assembly-options` block to your store's product page template.
-2. According to your store's scenario, declare the desired props.
+1. Add `product-customizer` app to your theme's dependencies in the `manifest.json`, for example:
+
+```diff
+  dependencies: {
++   "vtex.product-customizer": "2.x"
+ }
+```
+
+Now, you are able to use all blocks exported by the `product-customizer` app. Check out the full list below:
+
+| Block name     | Description                                     |
+| -------------- | ----------------------------------------------- |
+| `product-assembly-options` | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red)  Top level block responsible for displaying the product customizer default component. Use this block's children list to define which attachment data you want to display for users.   |
+| `assembly-option-input-values` | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) Displays a checkbox or a dropdown list field so users can choose the desired attachment for their products. |
+| `assembly-option-item-image` | Renders the attachment image. | 
+| `assembly-option-item-quantity-selector` | Renders a quantity selector. | 
+| `assembly-option-item-name` | Renders the attachment name. | 
+| `assembly-option-item-price` | Renders the attachment price (if any). |
+| `assembly-option-item-customize` | ???? Renders the attachment customization options. | 
```

Comment:
Renders a button Customize that when clicked opens a modal to customize the item's child assembly options if it has any.